spec/migrations/20160119125950_add_created_on_for_container_entities_spec.rb

```diff
@@ -0,0 +1,82 @@
+require_migration
+
+def stub_name_for(table)
+  "#{table}_stub"
+end
+
+def stub_for(table)
+  send(stub_name_for(table))
+end
+
+describe AddCreatedOnForContainerEntities do
+  CONTAINER_TABLES = [:container_nodes, :container_projects, :container_services, :container_routes, :container_groups,
+                      :container_replicators, :container_quotas, :container_builds, :container_build_pods,
+                      :container_limits, :container_volumes].freeze
```

Comment:
great, I didn't know about it before (just with `table_name` for our case)

fyi @Ladas